internal/k8sinternal/client.go

```diff
@@ -65,6 +65,9 @@ func NewKubeClientLocal(configPath string) (KubeClient, error) {
 		return nil, err
 	}
 
+	// Ignore warnings from kubeclient as they are expected to be reported by the deprecatedapi auditor.
+	kubeconfig.WarningHandler = rest.NoWarnings{}
+
```

Comment:
This  will hide warning messages such as the following:
```
W0608 12:04:51.010357   30221 warnings.go:70] v1 ComponentStatus is deprecated in v1.19+
W0608 12:04:54.355517   30221 warnings.go:70] extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
```

Kubeaudit has to use the deprecated APIs to include them in the deprecatedapis reports.  
See https://kubernetes.io/blog/2020/09/03/warnings/#customize-client-handling